prov/gni/test/cntr.c

```diff
@@ -98,6 +98,7 @@ static inline void cntr_setup_eps(void)
 	hints->mode = ~0;
 
 	hints->fabric_attr->name = strdup("gni");
+	hints->caps = GNIX_EP_RDM_PRIMARY_CAPS;
```

Comment:
So would we consider this a problem with the test program or with autoprogress or...?